### ISSUE TITLE
Language won't change from Japanese to English (missing en_US locale?)

### ISSUE THREAD

**Issue by jezs** [OpenShot/openshot-qt#749]
**System Details:**
* Operating System / Distro: **Windows 7 Professional 64-bit** 
* OpenShot Version: **2.3.4**
* _Please attach log files if crash: **N/A**

**Issue Description and steps to reproduce:**

I just freshly installed OpenShot. Using the language selection UI closed in issue #97 , I changed my language preference from Default to American English (en_US). However, when I restarted the application, the menu titles were still in Japanese. It did not change. I tried setting it to Canadian English (ca_US) and it was successfully changed.

My suspicion is that this is due to a missing locale under the "C:\Program Files\OpenShot Video Editor\locale" folder.

To support this, I checked the log openshot-qt.log and found that it loads locales from my "C:\Program Files\OpenShot Video Editor\locale" folder.

```
      launch:INFO ------------------------------------------------
      launch:INFO    OpenShot (version 2.3.4)
      launch:INFO ------------------------------------------------
         app:INFO openshot-qt version: 2.3.4
         app:INFO libopenshot version: 0.1.7
         app:INFO platform: Windows-7-6.1.7601-SP1
         app:INFO processor: Intel64 Family 6 Model 69 Stepping 1, GenuineIntel
         app:INFO machine: AMD64
         app:INFO python version: 3.5.2
         app:INFO qt5 version: 5.6.1
         app:INFO pyqt5 version: 5.6
    language:INFO Attempting to load qt_ja_JP in 'C:/Program Files/OpenShot Video Editor/translations'
    language:INFO Attempting to load qt_ja in 'C:/Program Files/OpenShot Video Editor/translations'
    language:INFO Attempting to load ja_JP\LC_MESSAGES\OpenShot in 'C:\Program Files\OpenShot Video Editor\locale'
    language:INFO Attempting to load ja\LC_MESSAGES\OpenShot in 'C:\Program Files\OpenShot Video Editor\locale'
    language:INFO Successfully loaded ja\LC_MESSAGES\OpenShot in 'C:\Program Files\OpenShot Video Editor\locale'
    language:INFO Qt Detected Languages: ['ja-JP']
    language:INFO LANG Environment Variable: ja_JP
    language:INFO LOCALE Environment Variable: ja_JP

```

I checked the available locales under  "C:\Program Files\OpenShot Video Editor\locale" and there was no en_US. Perhaps under the hoods, it uses the locale code en_US to load translations from the locale folder.

For your perusal.

**Comment by DylanC** [comment]
Dup of #331

**Comment by peanutbutterandcrackers** [comment]
A commit has recently been made to the source code that should fix this. Could you please try out the latest [daily build](https://github.com/OpenShot/openshot-qt/releases/tag/daily) and see if this problem still persists or not? Thank you! :)

**Comment by DylanC** [comment]
@jezs - Confirmed as fixed in #331. Please can you confirm this works for you too.

**Comment by DylanC** [comment]
Confirmed as fixed in #331.
